ocaml/xapi/xapi_pusb.ml

```diff
@@ -48,10 +48,21 @@ let scan_start ~__context usbs =
   List.filter (fun (rf, rc) -> USBSet.mem (extract_known_usb_info rc) (USBSet.diff known_usb local_usb)) (Db.PUSB.get_all_records ~__context)
   |> List.iter (fun (self, _) ->
                   let usb_group = Db.PUSB.get_USB_group ~__context ~self in
-                  Db.PUSB.destroy ~__context ~self;
                   let vusbs = Db.USB_group.get_VUSBs ~__context ~self:usb_group in
+                  List.iter (fun vusb ->
+                    let currently_attached = Db.VUSB.get_currently_attached ~__context ~self:vusb in
+                    if currently_attached then
+                      try
+                        Helpers.call_api_functions ~__context (fun rpc session_id ->
+                          Client.Client.VUSB.unplug rpc session_id vusb)
+                      with _ ->
```

Comment:
Should we also add the specific error in the error message?

Reply:
Hold on, this doesn't do what I suggested - I just wanted to log there had been a problem unplugging the VUSB and then move on to the next PUSB to remove.

Reply:
totally right.

Reply:
Will error be ok or just give debug log ?